python/tests/test_framework/slate/data_generation.py

```diff
@@ -0,0 +1,72 @@
+import random
+import os
+from test_helper import get_function_object
+
+script_directory = os.path.dirname(os.path.realpath(__file__))
+random.seed(10)
+
+
+def generate_slate_data(
+    num_examples,
+    reward_function,
+    logging_policy,
+    action_space,
+    num_context=1,
```

Comment:
num_context is conflicting with context_name. Can we allow context_name to be a list and infer no_context from it?